portability-core/src/main/java/org/dataportabilityproject/shared/settings/CommonSettings.java

```diff
@@ -60,9 +60,10 @@ public CommonSettings(
       // locally and connecting to GCP
       String googleProjectId = System.getenv("GOOGLE_PROJECT_ID");
```

Comment:
Do we read this twice from the environment variable now?

Reply:
I just made ProjectId a singleton in the module to avoid that

Reply:
Looking at it closer, maybe we should avoid reading a Google cloud specific value, wether in environment or singleton,  in the CommonSettings. Can we move this logic elsewhere?

Reply:
good idea, moved it to GoogleCloudModule and got rid of the extra env variable read